src/lib/Scenes/Collection/Screens/__tests__/__snapshots__/CollectionArtworkPreview-tests.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders properly 1`] = `"<view mb=\\"4\\" width=\\"100%\\"></view>"`;
```

Comment:
This looks different than before, but I still don't think it's capturing what you intended it to capture.

Reply:
So this new snapshot is testing the Container and that the relay mock query generates the expected underlying view component. If you compare a similar snapshot in the Show Artworks preview, it's compiling to the same underlying view: https://github.com/artsy/emission/blob/344542288c6b2dbe85f3758224dae88cba853c10/src/lib/Components/Show/__tests__/__snapshots__/ShowArtworksPreview-tests.tsx.snap#L3

What do you think?

Reply:
I don't think it's a test worth keeping, if that's what the snapshot looks like. That's not going to catch any changes to the underlying component. I'd ditch the test, or find a way to test the underlying component itself.

Reply:
I'm going to disable the test for now and follow up with @ds300 or @ashfurrow when they return next week from the break. In the meantime I think this PR is safe to merge!